packages/api/public/spec/paths/languages/language-members.yml

```diff
@@ -34,7 +34,7 @@ get:
 post:
   summary: Invite a new member to a language.
```

Comment:
I noticed that this route is missing a tag, so it shows up under a "default" section in the API explorer.

```suggestion
  summary: Invite a new member to a language.
  tags:
     - Languages
```

Reply:
Done.